src/main/java/duke/dukeobject/Expense.java

```diff
@@ -219,6 +219,16 @@ public LocalDateTime getTime() {
         return time;
     }
 
+    public String getTagsString() {
+        StringJoiner stringJoiner = new StringJoiner(" ");
+        if(tags.isEmpty()) {
+            return "";
+        }else{
+            stringJoiner.add("Tags: " + String.join(" ", tags));
+        }
+        return stringJoiner.toString();
+    }
+
```

Comment:
should be part of DukeItem, as it is the class that defines tags

Reply:
Fixed thanks!